FreeRTOS/FreeRTOS-Kernel
# Update task running state macro and type

Update task running state macro and types.

Description
-----------
* Remove type TaskRunning_t use BaseType_t for TaskRunning_t directly.
* Rename taskTASK_YIELDING to taskSCHEDULED_TO_YIELD.

Test Steps
-----------


Checklist:
----------



Related Issue
-----------
Address #750 comment in line 263, 266, 268.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
